docs/03-✅-backend-principles.md

```diff
@@ -0,0 +1,13 @@
+# Backend Principles
```

Comment:
should we also add in that there are zip files and s3 buckets and they are a 1:1 match content wise?

Reply:
yeah that's definitely worth adding